### ISSUE TITLE
How can i get search term when user typing on search select with searchable=true

### ISSUE THREAD

**Issue by mqnoy** [rizzui/rizzui#78]
i'm trying to get user input, but no luck

```
const searchProps: InputHTMLAttributes<HTMLInputElement> = { onChange: (e) => { console.log(e.target.value); }, };
<Select 
searchProps={searchProps}
/>
```

regarding the code:

https://github.com/rizzui/rizzui/blob/a683db98a992ab2169da239d5375b3631f311984/packages/ui/src/components/select/select.tsx#L424

**Comment by md-nahid** [comment]
Hi @mqnoy we have fixed it. It's in testing stage. Soon we will release it.

**Comment by mqnoy** [comment]
I think on #77  i seen added callback for this issue ,  good work 🙂👍


Thanks

**Comment by idris95** [comment]
Hi @mqnoy,  we've released **rizzui@0.8.6**. Please check it out.
